variants/t-echo/variant.h

```diff
@@ -168,6 +168,7 @@ External serial flash WP25R1635FZUIL0
  * GPS pins
  */
 
+#define PIN_GPS_RESET (32 + 5) // An output to reset L76K GPS. As per datasheet, low for > 100ms will reset the L76K
 #define PIN_GPS_WAKE (32 + 2) // An output to wake GPS, low means allow sleep, high means force wake
```

Comment:
Can you describe the difference in behaviour of thechip with and without this code? Cause it's behaving exactly the same for me when i enable this.